checkstyle-config/src/main/resources/graphicsfuzz/checkstyle.xml

```diff
@@ -74,8 +74,8 @@ limitations under the License.
             <property name="tokens" value="LITERAL_TRY, LITERAL_CATCH, LITERAL_FINALLY, LITERAL_IF, LITERAL_ELSE, LITERAL_DO"/>
         </module>
         <module name="RightCurly">
-            <property name="id" value="RightCurlyAlone"/>
-            <property name="option" value="alone"/>
+            <property name="id" value="RightCurlySame"/>
+            <property name="option" value="same"/>
```

Comment:
I'm not sure what effect these changes have - are they in line with Google style?

Reply:
Yes. There might even be more changes to the Google style since our snapshot, but this was enough to get our code passing.